includes/admin/class-wp-job-manager-addons-landing-page.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * File WP_Job_Manager_Addons_Landing_Page class.
+ *
+ * @package wp-job-manager
+ * @since $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Landing pages for Applications and Resumes add-ons.
+ *
+ * @since $$next-version$$
+ */
+class WP_Job_Manager_Addons_Landing_Page {
+
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var self
+	 * @since  $$next-version$$
+	 */
+	private static $instance = null;
+
+	/**
+	 * Singleton accessor.
+	 *
+	 * @since  $$next-version$$
+	 * @static
+	 * @return self Main instance.
+	 */
+	public static function instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Initialize class for landing pages.
+	 *
+	 * @since $$next-version$$
+	 */
+	public function __construct() {
+		add_action( 'admin_menu', [ $this, 'add_menu_items' ], 12 );
+		WP_Job_Manager::register_style( 'job_manager_admin_landing_css', 'css/admin-landing.css', [ 'job_manager_brand' ] );
```

Comment:
I think we should register these styles in `admin_enqueue_scripts`.

```suggestion
					add_action( 'admin_enqueue_scripts', [ $this, 'register_styles' ] );
	}

	/**
	 * Register styles for landing pages.
	 *
	 * @access public
	 * @since $$next-version$$
	 */
	public function register_styles() {
		WP_Job_Manager::register_style( 'job_manager_admin_landing_css', 'css/admin-landing.css', [ 'job_manager_brand' ] );
```

Reply:
👍  Added this and your other suggestions in [29b8385](https://github.com/Automattic/WP-Job-Manager/pull/2614/commits/29b83855d5f78bc2b2aa333e88e65ae1972003cd)